.github/workflows/release.yml

```diff
@@ -18,27 +17,34 @@ jobs:
         with:
           python-version: '3.13'
 
-      - name: Check package version matches tag
+      # Install dependencies for Poetry and the package
+      - name: Install dependencies
         run: |
-          pkgver=$(python -c "import autocorpus; print(autocorpus.__version__, end='')")
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install --no-root  # Install dependencies without installing the package
```

Comment:
This "Install Dependencies" step should be redundant because they are installed in the `setup` composite action used above (which is defined in [.github/actions/setup/action.yml](https://github.com/omicsNLP/Auto-CORPus/blob/main/.github/actions/setup/action.yml))

Reply:
And, in fact, it might cause a problem because it's re-installing poetry, which is already installed)

Reply:
removed 👍